qp_shotgun/qc_trim/tests/test_qc_trim.py

```diff
@@ -180,9 +181,9 @@ def test_generate_qc_trim_analysis_commands_forward_reverse(self):
         self._clean_up_files.append(fp)
 
         exp_cmd = [
-			'atropos --threads 4 --A GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT --adapter GATCGGAAGAGCACACGTCTGAACTCCAGTCAC --max-n 80 --minimum-length 80 --nextseq-trim False --pair-filter any --quality-cutoff 15 --trim-n True -o output/fastq/s1.fastq -p output/fastq/s1.R2.fastq -pe1 fastq/s1.fastq -pe2 fastq/s1.R2.fastq',
```

Comment:
Note that I'm changing up the output naming here... We want to modify f_fp and r_fp to create unique output filenames, I think.